readthedocs/core/models.py

```diff
@@ -51,6 +51,13 @@ class UserProfile(TimeStampedModel):
         default=False,
         null=True,
     )
+    # NOTE: this is a temporary field that we can remove after October 16, 2023
+    # See https://blog.readthedocs.com/build-image-config-deprecated/
```

Comment:
The blog file is: use-build-os-config.rst

Reply:
```suggestion
    # See https://blog.readthedocs.com/use-build-os-config/
```